.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -49,33 +64,39 @@ This checklist is focused on a classic Jahia repository (module, app)
 - [ ] 🟠 No warning are presents in the browser console when using the app
+- [ ] 🙏 Remove unreachable code
 ### Java
 - [ ] 🔝 Java dependencies are explicitly declared in the module's pom.xml
 - [ ] 🔝 Spring is not used in the module
-- [ ] 🔝 No warnings or errors are present when building the module locally or on GitHub Actions
-- [ ] 🟠 No code smell on [Sonarqube](https://sonarqube.jahia.com/projects) for the module
 ### Security
-- [ ] 🔝 Our security lead confirmed there are no known security vulnerabilities affecting this codebase
-### QA
+- [ ] 🔝 Review the [security vulnerabilities](https://dependency-track-prod.jahia.com/) affecting this codebase and discuss with the Security lead before taking action (Create a [SECURITY](https://support.jahia.com/browse/SECURITY) ticket, Close as false-positive, etc.)
+### QA / Automated Tests
+- [ ] 🚨 The codebase is compatible with the latest release of Jahia
 - [ ] 🔝 Automated tests are using jahia-cypress for all utils functions
 - [ ] 🔝 The test framework is using page-object models published by other modules
 - [ ] 🔝 The test framework is publishing its own page-object models for use by others
+- [ ] 🟠 A manual-run workflow is available
+- [ ] 🟠 Instructions and [test cases](https://jahia.testrail.net/index.php?/dashboard) are available to document how a release should be tested
 - [ ] 🙏 Automated tests are using a recent version of Cypress
 - [ ] 🙏 Automated tests are only relying on supported modules
 ### CI/CD
 - [ ] 🔝 The build and the release workflows use the JDK 11 image (only if Jahia Parent is set to 8.2.0.0+)
-- [ ] 🔝 GitHub Actions (nightlys and other workflows) are executed without warnings (such as depreciations)
+- [ ] 🔝 GitHub Actions (nightlys and other workflows) are executed without warnings nor errors (such as depreciations, failed tests, ...)
 - [ ] 🙏 The latest version of the actions are used (including jahia-modules-action)
+- [ ] 🙏 GitHub Actions [reusable workflows](https://github.com/Jahia/jahia-modules-action/tree/main/.github/workflows) are used
 ### Documentation
+- [ ] 🔝 Readme.md is up-to-date (module purpose, technical details, configuration steps)
+- [ ] 🟠 A tech roadmap is available 
 - [ ] 🙏 Module's documentation available on the academy is up-to-date
+### Issues
+- [ ] 🟠 If the repository is public, issues/pull requests from the community have been reviewed and answered, if answer was not possible, a PM/DM was notified.
 ### GitHub
 - [ ] 🟠 [Branch protection](https://confluence.jahia.com/display/PR/GitHub+%28Product%29+-+Ref+ISPOL08.A14025#GitHub(Product)RefISPOL08.A14025-Branchprotection) is enabled for the repository
 - [ ] **Automatically delete head branches** is selected in **Settings**
-- [ ] 🙏 The repository contains a README.md file
 - [ ] 🙏 Repository topics match are populated (at a minimum: "product" and "supported")
 - [ ] 🙏 Stale branches or branches older than 2 years (non-maintenance branches) have been removed
```

Comment:
suggestion: Github settings - check to make sure merge is disabled and squash is enabled

Reply:
@rknj I wonder if this should not be a pointer to the GitHub repo settings guideline to avoid having this information in multiple places. i.e. one single check "The repo configuration complies with Jahia repository settings guidelines (LINK)".

Reply:
I guess you're talking about https://jahia-confluence.atlassian.net/wiki/spaces/PR/pages/2067858/GitHub+Product+-+Ref+ISPOL08.A14025#Repository-configuration ?
It might be worth moving it to Github (like in the .github repo?)

Reply:
that' a good question, but I think that should be an effort for the entire product space, if we split too much between confluence and GitHub, it's going to be difficult to figure out what is where.